logs.py

```diff
@@ -1,9 +1,35 @@
+from pubnub import Pubnub
+
 from .base_request import BaseRequest
 from .resources import Message
+from .models.config import Config
+
+class Logs(object):
+	def __init__(self):
+		self.config = Config()
+		
+
+	def subscribe(self, uuid, callback, error):
+		pubnub_key = self.config.get_pubnub_keys()
+		if not hasattr(self, 'pubnub'):
```

Comment:
Why bother with hasattr -- just set self.pubnub in __init__ and check here for "if self.pubnub is None"